zerver/tests/test_typing.py

```diff
@@ -96,9 +102,13 @@ def test_multiple_recipients(self):
 
         event = events[0]['event']
         event_recipient_emails = set(user['email'] for user in event['recipients'])
+        event_user_ids = set(events[0]['users'])
+        event_recipient_user_ids = set(user['user_id'] for user in event['recipients'])
 
+        self.assertTrue(expected_recipient_ids == event_recipient_user_ids)
```

Comment:
We mostly use assertEqual for these kind of checks.  It's mostly a consistency thing, but you also get slightly more useful output if the assertion fails.